source/ProcessManager.Components/EnqueueActorMessages/EnqueueActorMessagesSyncClient.cs

```diff
@@ -12,16 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.Text;
+using System.Text.Json;
 using Energinet.DataHub.ProcessManager.Components.Abstractions.EnqueueActorMessages;
+using Energinet.DataHub.ProcessManager.Components.Extensions.DependencyInjection;
 
 namespace Energinet.DataHub.ProcessManager.Components.EnqueueActorMessages;
 
-internal class EnqueueActorMessagesSyncClient : IEnqueueActorMessagesSyncClient
+internal class EnqueueActorMessagesSyncClient(
+    IHttpClientFactory httpClientFactory) : IEnqueueActorMessagesSyncClient
 {
+    // TODO: should we pass this in via app setting instead?
+    public const string EdiEndpointPrefix = "api/enqueue/";
```

Comment:
```suggestion
    public const string EdiEndpointPrefix = "/api/enqueue/";
```

I'm not sure if it matters, but in the `ProcessManagerClient` the URL's are prefixed with `/`. This means that the path starts at the root of the base URL.
![image](https://github.com/user-attachments/assets/d9a74575-771e-4eac-88d7-9f040e4cc7ee)